src/search-jobs.js

```diff
@@ -2,87 +2,109 @@ const Apify = require('apify');
 const cheerio = require('cheerio');
 
 const { log } = Apify.utils;
-const { BASE_URL, REQUEST_HEADERS } = require('./consts');
 
-const searchJobs = async (query, location, maxResults, proxyUrl) => {
-    const searchEndpoint = '/Job/jobs.htm';
+const { BASE_URL,
+} = require('./consts');
 
-    // global variable for loaded cheerio content to keep jQuery-alike syntax
+const searchJobs = async ({ request }, requestQueue, proxyUrl) => {
```

Comment:
you can use the proxyConfiguration inside and call `proxyConfiguration.newUrl(session.id)` from here